cdist/conf/type/__line/man.rst

```diff
@@ -40,6 +40,13 @@ file
     If supplied, use this as the destination file.
     Otherwise the object_id is used.
 
+MESSAGES
+--------
+line added
+        The line was added.
+
+line removed
+        The line was removed.
```

Comment:
I suggest to omit "line", as you will grep for __line as a prefix anyway - just added/removed should be fine